test/sequential/test-inspector-runtime-evaluate-with-timeout.js

```diff
@@ -16,6 +16,8 @@ common.skipIfInspectorDisabled();
     expression: 'for(;;);',
     timeout: 0
   }).catch((e) => e);
-  strictEqual(result.message, 'Execution was terminated');
+  strictEqual(result.code, 'ERR_INSPECTOR_COMMAND');
+  strictEqual(result.message,
+              'Inspector error -32000: Execution was terminated');
```

Comment:
This could be changed to:

```js
await assert.rejects(
  session.post('Runtime.evaluate', {
    expression: 'for(;;);',
    timeout: 0
  }),
  {
    code: 'ERR_INSPECTOR_COMMAND',
    message: 'Inspector error -32000: Execution was terminated'
  }
);
```